### DISCUSSION TITLE
Add Inertia as New Feature to Connect Between IntentJS Into Any SPA System Like Laravel + InertiaJS

### DISCUSSION THREAD

**Discussion by cakadi190** [intentjs/core#39] [Ideas]
Hi, since following your IntentJS Project which looks like Laravel, I would like to request one feature which is important for Fullstack Developers. Namely Integeration with the same InertiaJS as in Laravel. Hopefully it will be realized and published soon. Thank you

**Comment by vinayak25** [comment]
Hello @cakadi190 I have been thinking about integrating the intertiajs, but I still have my own doubts for the SSR stuff just like how NextJS works.

Maybe, I will publish the inertiajs integration first, and then start working on the SSR part.

**Reply by cakadi190** [reply]
Ah, I see @vinayak25. I suggest for temporary solution, you can use "blade" like templating and pass it from inertia to "blade" like templating which is same as like `Inertia::render()->withViewData()` code on [this reference](https://inertiajs.com/responses).

I will wait for the good news from this project.
